types/qrcode-svg/tslint.json

```diff
@@ -0,0 +1,6 @@
+{
+    "extends": "dtslint/dt.json",
+    "rules": {
+        "unified-signatures": false
```

Comment:
This file should only `"extends": "dtslint/dt.json"`

Reply:
Done